test-data/unit/check-functions.test

```diff
@@ -737,6 +737,14 @@ class A:
 main:6: error: Incompatible types in assignment (expression has type "int", variable has type "A")
 main:8: error: Argument 1 to "g" has incompatible type "A"; expected "int"
 
+[case testNestedFunctionInMethodWithTooFewArgumentsInTypeComment]
```

Comment:
Perhaps you could also add a test for a few other configurations of the stack, e.g. a method inside a class inside a method (i.e. `['C', 'F', 'C', 'F']`)?